src/main.h

```diff
@@ -44,13 +44,13 @@ static const bool DEFAULT_ALERTS = true;
 /** Default for -maxorphantx, maximum number of orphan transactions kept in memory */
 static const unsigned int DEFAULT_MAX_ORPHAN_TRANSACTIONS = 100;
 /** Default for -limitancestorcount, max number of in-mempool ancestors */
-static const unsigned int DEFAULT_ANCESTOR_LIMIT = 100;
+static const unsigned int DEFAULT_ANCESTOR_LIMIT = 25;
 /** Default for -limitancestorsize, maximum kilobytes of tx + all in-mempool ancestors */
-static const unsigned int DEFAULT_ANCESTOR_SIZE_LIMIT = 900;
+static const unsigned int DEFAULT_ANCESTOR_SIZE_LIMIT = 100;
```

Comment:
I'm not sure that's really a problem -- maximum size transactions seem sufficiently rare that perhaps people who want to use CPFP can just limit their transaction sizes to, say, 99kb?  Is there much harm done in that?

If this is really a problem we can just make this limit 101k instead.